examples/counter_out/write_dig_pulse.py

```diff
@@ -0,0 +1,19 @@
+"""Example for generating digital pulse.
+
+This example demonstrates how to generate a single digital pulse
+from a Counter Output Channel. The Initial Delay, High Time, Low
+Time, and Idle State are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import Level
+
+with nidaqmx.Task() as task:
+    channel = task.co_channels.add_co_pulse_chan_time(
+        "Dev1/ctr1", idle_state=Level.LOW, initial_delay=0.0, low_time=0.01, high_time=0.01
```

Comment:
I like the C example's defaults better because it's slow enough to visually see the pulse without an oscilloscope or ctr/timer (like with an LED, logic probe, or static DI loopback). (The pulse timing is also reflected by the duration of the DAQmxWaitUntilDone call in NI IO Trace.)

```
DAQmxErrChk (DAQmxCreateCOPulseChanTime(taskHandle,"Dev1/ctr0","",DAQmx_Val_Seconds,DAQmx_Val_Low,1.00,0.50,1.00));
```

Reply:
Noted, the low time and high time is updated as per C example.